### PR TITLE
Updated Gmail Trace Logs Source doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md [hunk 1]
```diff
@@ -18,52 +17,133 @@ You'll need to use our [Google BigQuery source](/docs/send-data/hosted-collector
...
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. **Project ID**. Enter the unique identifier number. You can find this from the Google Cloud Console.
+1. **Dataset ID**. Enter the The Dataset ID is the project-wise unique identifier for your dataset.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md:77]:
```suggestion
1. **Dataset ID**. Enter the ID. The Dataset ID is the project-wise unique identifier for your dataset. 
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md [hunk 2]
```diff
@@ -18,52 +17,133 @@ You'll need to use our [Google BigQuery source](/docs/send-data/hosted-collector
...
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. **Project ID**. Enter the unique identifier number. You can find this from the Google Cloud Console.
+1. **Dataset ID**. Enter the The Dataset ID is the project-wise unique identifier for your dataset. 
+1. **Data Location**. Enter the location of DataSet which is set while creating Dataset in BigQuery.
+1. **Private Key**. Enter the private key of Service Account JSON, it is a security key which is required for authentication. You can find this from the Google Cloud Console.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source.md:79]:
```suggestion
1. **Private Key**. Enter the private key of Service Account JSON. This is a security key which is required for authentication. You can find this from the Google Cloud Console.
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
This is very good, @JV0812. I found only a few little things.

**Comment by JV0812** [comment]
Thanks John. I have incorporated your comments.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**